content/secrets/12-walkthrough-step-09/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Inject Secure Parameters"
+chapter: false
+weight: 35
+---
+
+
+When working with secrets in an AWS infrastructure workload, you have the option to use AWS System Manager Parameter Store.  
+
+AWS Systems Manager Parameter Store provides secure, hierarchical storage for configuration data management and secrets management. You can store data such as passwords, database strings, Amazon Machine Image (AMI) IDs, and license codes as parameter values. 
+
+You can store values as plain text or encrypted data. You can reference Systems Manager parameters in your scripts, commands, SSM documents, and configuration and automation workflows by using the unique name that you specified when you created the parameter.
+
+As a best practice, you should generally opt to use Secrets Manager over Parameter store for secure credentials infrequently accessed.   
+
+{{%expand "Click here to expand feature comparison" %}}
```

Comment:
I would prefer to lead the user to the docs for that feature comparison (assuming it's available). This way, if/when things change with one or both of the services, we don't have stale content.

Reply:
I didnt find this in a handy table anywhere, i built it from reading the docs.   Ill search to see if its documented anywhere official in this format.